processor/blockprocessor/block_processor.go

```diff
@@ -195,14 +234,16 @@ func prepareEvalResources(l *indexerledger.LedgerForEvaluator, block *bookkeepin
 }
 
 // Preload asset and app creators.
-func prepareCreators(l *indexerledger.LedgerForEvaluator, payset transactions.Payset) (map[basics.AssetIndex]ledger.FoundAddress, map[basics.AppIndex]ledger.FoundAddress, error) {
+func prepareCreators(lfe *indexerledger.LedgerForEvaluator, payset transactions.Payset) (map[basics.AssetIndex]ledger.FoundAddress, map[basics.AppIndex]ledger.FoundAddress, error) {
 	assetsReq, appsReq := accounting.MakePreloadCreatorsRequest(payset)
-
-	assets, err := l.GetAssetCreator(assetsReq)
+	assets, err := lfe.GetAssetCreator(assetsReq)
+	if err != nil {
```

Comment:
I didn't see a good reason not to percolate `err` up for `GetAssetCreator()`. Is there a good reason this wasn't previously done?